packages/dd-trace/test/priority_sampler.spec.js

```diff
@@ -143,7 +150,15 @@ describe('PrioritySampler', () => {
     })
 
     it('should support a global sample rate', () => {
-      prioritySampler = new PrioritySampler('test', { sampleRate: 0.5 })
+      config.configure({
+        env: 'test',
+        experimental: {
+          sampler: {
```

Comment:
This is no longer experimental, so probably best to use the new `ingestion` config.

Reply:
Hmm, yeah, this is a merge shenanigan, I think. Will fix.

Reply:
Wasn't a merge shenanigan, but copypasta. Fixed.